examples/hosts/app-integration/external-data/src/model/taskList.ts

```diff
@@ -542,13 +542,24 @@ export class BaseDocument extends DataObject implements IBaseDocument {
 		return this.taskListCollection.get(id);
 	};
 
+	public readonly getLeader = (): string | undefined => {
```

Comment:
Nit: these are implementations of interface methods, right? If so, might be worth adding `@inheritDoc` comments so there is a pointer to the docs.